test/unit/foreman/cron_test.rb

```diff
@@ -0,0 +1,85 @@
+require 'test_helper'
+require 'foreman/cron'
+
+class Foreman::CronTest < ActiveSupport::TestCase
+  setup do
+    Foreman::Cron.instance_variable_set(:@extra_tasks, nil)
+    Foreman::Cron.instance_variable_set(:@logger, nil)
+
+    @mock_logger = mock('logger')
+    @mock_logger.stubs(:info)
+    @mock_logger.stubs(:debug)
```

Comment:
This seems to break tests, even tho I don't really understand why.

Reply:
yeah, simplified it now, earlier it was mocked `Rails.logger` can cause the test break